src/index.js

```diff
@@ -1,12 +1,39 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
 import App from './App';
+import rootReducer from './reducers/index';
 import reportWebVitals from './reportWebVitals';
 
+const bookReducer = [
+  {
+    id: Math.random().toFixed(2),
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Principles', category: 'Learning'}
```